packages/ui/src/components/editor/input/Vector3/index.tsx

```diff
@@ -93,7 +96,13 @@ export const Vector3Input = ({
   onRelease,
   ...rest
 }: Vector3InputProp) => {
-  const uniformEnabled = useHookstate(uniformScaling)
+  const id = uniqueId++
+  const uniformEnabled = useHookstate(false)
```

Comment:
We are using ids for accessibility purposes which should remain constant. Due to re-rendering, ids will change. React provides a [useId](https://react.dev/reference/react/useId) hook for generating unique ids.

![image](https://github.com/EtherealEngine/etherealengine/assets/55396651/419292e1-50e8-4e51-838c-3a2f65af7cc7)

In this case, we can wrap the input inside label somewhat like this:
```tsx
            <label title="Uniform Scale" htmlFor={checkboxId}>
              <input
              className="hidden"
              id={checkboxId}
              type="checkbox"
              checked={uniformEnabled.value}
              onChange={onToggleUniform}
              />
              <Icon type={uniformEnabled.value ? 'Link' : 'LinkOff'} />
            </label>
```

I will replace this with a tailwind button with the icon set conditionally

Reply:
Cool thanks, I just copied it from the old implementation that was lost